<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Create_surface</class>
 <widget class="QDockWidget" name="Create_surface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create and Deform Surface</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Surface Generation</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Delta Leader:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="leaderSpinBox">
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Delta Generator:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="generatorSpinBox"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="createSurfaceButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Create Surface</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Polyline Creation</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="newPolylineButton">
               <property name="text">
                <string>New Generator</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="finishButton">
          <property name="text">
           <string>Finish</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="cancelButton">
          <property name="text">
           <string>Cancel Last Insertion</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
